.gitignore

```diff
@@ -4,3 +4,5 @@
 *.sw?
 *.bak
 *.rej
+venv/
```

Comment:
why is there a `venv` here?

Reply:
That was @mbautin, commit 963b3d8.  I was using a virtual environment.